react/src/lib/components/DeckGLMap/layers/intersection/unfoldedGeoJsonLayer.ts

```diff
@@ -23,64 +21,51 @@ function getUnfoldedPath(object: Feature): Position3D[] {
     });
     const vAbscissa = zip(a, [...a].fill(planeY), z);
 
-    return vAbscissa as Position3D[];
+    return vAbscissa as Position[];
```

Comment:
For here, can we add a type check to make the code more safe rather than using as to enforce the type.